gen3-cli/tests/integration/backend/test/test_generate.py

```diff
@@ -0,0 +1,47 @@
+import os
+import shutil
+import tempfile
+from cot.backend.test.generate import run as generate_test_backend
+from .conftest import DATA_DIR
+
+
+def test():
+    with tempfile.TemporaryDirectory() as dir:
```

Comment:
`dir` seems to be a pythonic word, so may be it's better to select some different name without losing the readability (like tmp_dir)

Reply:
Ye, you're right